Lib/ctypes/__init__.py

```diff
@@ -356,15 +356,15 @@ def __init__(self, name, mode=DEFAULT_MODE, handle=None,
                archive(member) syntax for dlopen(), and the mode is adjusted.
                Otherwise, name is presented to dlopen() as a file argument.
             """
-            if name and name.endswith(")") and ".a(" in name:
```

Comment:
It would be cleaner to just reassign `name to self._name` and not change these.

Reply:
Done.